examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 xpokor22
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+#include <ir_LG.h>
+
+IRsend irsend(14);  // An IR LED is controlled by GPIO pin 14 (D5)
+
+// 0 : TOWER
+// 1 : WALL
+const unsigned int AC_TYPE  = 1;
```

Comment:
You probably should make this a #define rather than a const. But that's a nitpick. Not a required change.